shared/secrets_masker/tests/secrets_masker/test_secrets_masker.py

```diff
@@ -243,10 +253,15 @@ def test_masking_in_explicit_context_exceptions(self, logger, caplog):
         ],
     )
     def test_mask_secret(self, name, value, expected_mask):
-        filt = SecretsMasker()
-        filt.add_mask(value, name)
+        from airflow_shared.secrets_masker.secrets_masker import _secrets_masker
+        filt = _secrets_masker()
+        configure_secrets_masker_for_test(filt)
```

Comment:
If we are configuring it, why do we need to use the global `_secrets_masker()` instead of creating an object only used for this test?

Reply:
Ive handled it better now